libs/gi/ui/src/components/artifact/ArtifactCard.tsx

```diff
@@ -483,10 +496,12 @@ function SubstatDisplay({
   stat,
   effFilter,
   rarity,
+  isActiveStat = true,
 }: {
   stat: ICachedSubstat
   effFilter: Set<SubstatKey>
   rarity: ArtifactRarity
+  isActiveStat?: boolean
 }) {
   const { t: tk } = useTranslation('statKey_gen')
```

Comment:
```suggestion
  const { t: tk } = useTranslation(['statKey_gen', 'ui'])
```
assuming you put the unactivated translation in ui.json